metadata.json

```diff
@@ -1,14 +1,10 @@
 {
   "type": "model",
-  "uid": "minimalModel",
-  "name": "Mininal Model",
+  "uid": "populationGrowthModel",
+  "name": "Population Growth Model",
```

Comment:
in the stable version, `name` was renamed to `title` and we now need to assign a version number:
``` json
"version": 1,
"title": "Population Growth Model",
```